README.md

```diff
@@ -165,6 +223,50 @@ would become the following in beancount:
     2018-02-02 * "Train Brussels airport to city" ^2018-02-02-brussels-fosdem #debian
 
 
+### Comments
+
+Comments are supported.  Since beancount currently [doesn't allow
+full-line comments within
+transactions](https://bitbucket.org/blais/beancount/issues/143), these
+are stored as metadata.
+
+
+### Lots
+
+Lot costs and prices are supported, including per-unit and total lot
+costs.  Lot dates are converted to beancount but lot notes are not
+handled at the moment.
```

Comment:
file a feature request for this one too? (I don't know how important this is for the average user TBH)

Reply:
I think lot notes are quite rare but I'll look into it. I filed an issue.